### DISCUSSION TITLE
How to access jwt-payload from rodauth object?

### DISCUSSION THREAD

**Discussion by wuarmin** [jeremyevans/rodauth#438] [General]
Hey,

Why it is not possible to access the `jwt_payload` from a `rodauth`-object? Why `jwt_payload` is not a `public` method?

```ruby
class Userinfo < Auth::Action
  def handle(req, res)
    req.env["rodauth"] # NoMethodError: private method `jwt_payload' called for an instance of #<Class:0x00007fe14175e890>
  end
end
```
I don't want to load the account via `account!`, when it's not necessary, cause I just need an attribute from the `jwt_payload`.

Thanks

**Comment by jeremyevans** [comment]
In general, rodauth methods are private unless there is a reason for them to be public.  This gives more flexibility for future changes. If you'd like `jwt_payload` to be public, please submit a pull request with an example of why you want it to be public (the example given doesn't show why).

**Reply by wuarmin** [reply]
Oh, thanks, I understand.

Here's the reason, why I want jwt_payload to be public:

> I don't want to load the account via account!, when it's not necessary, cause I just need an attribute from the jwt_payload.
